Makefile

```diff
@@ -208,6 +212,16 @@ release: $(PY_PKG_INFO) $$($$RPM_PATH)
 docker-release: docker-rpm
 	docker run --rm -e ARTIFACTORY=$(ARTIFACTORY) $(DOCKER_TAG) /usr/bin/make release
 
+# See "release" target. The major difference is that this target uses the Box custom
+# rpm-to-artifactory tooling that is made available for signing and uploading rpms.
+.PHONY: release-signed
+release-signed: $(PY_PKG_INFO) $$($$RPM_PATH)
```

Comment:
Can we get rid of release and docker-release now?

Reply:
Also it is more useful for users outside of Box.